### ISSUE TITLE
Videos from this channel 'SonyMusicSouthVEVO' are not playing 

### ISSUE THREAD

**Issue by mdparthi** [nicholasguyett/Youtube-Audio-Player#1]
Hi , First of all I wanted to thank you for your great work on creating the audio player . Really great.

Also I noticed that songs from the channel id 'SonyMusicSouthVEVO'  are not playing. Can you please check whether it can be resolved , an example youtube id from the channel are "https://www.youtube.com/watch?v=Q2IiqLfUYgc" , "https://www.youtube.com/watch?v=fhdLTvl9a74"

Thanks for your work

**Comment by nicholasguyett** [comment]
I'm can't see exactly why at the moment, as I'm headed to church, but my
first guess would be that the videos aren't allowed to be embedded.  If you
unhide the hidden iframe and the YouTube player shows an error, then
there's nothing that can be done with my code.

On another note  how did you find my repo?  It's the first time I've seen
anyone but myself or co-workers interact with any of these, haha

**Comment by mdparthi** [comment]
Thanks Nicholas for your support. I found your repo on google search.

**Comment by nicholasguyett** [comment]
I think the problem is rooted in the fact that my repo comes with no instructions, haha.  The first example video you sent me worked fine for me when I set data-src on #player to the video id and when I used the tracklist (though I had to click on the track item before it would change).

The player first uses the id set in the data-src element and doesn't automatically pull from the tracklist rendered to the right.  The second one is giving me the error that the video is not available in my country.

Let me know if this fixes the issue for you.  I'll add a TODO to add some documentation for any future visitors.  Additionally, what search query did you use that my repo showed up on?  I'm curious what my ranking on the page was.
